### DISCUSSION TITLE
Where is Milvus c++ api ？

### DISCUSSION THREAD

**Discussion by 852511489** [milvus-io/milvus#6852] [Q&A and General discussion]
I looked through all the documents of mivus, but I couldn’t find the C++ api document.Where is the C++ api？

**Comment by yhmo** [comment]
For v1.x, the C++ sdk is in this path: https://github.com/milvus-io/milvus/tree/1.1/sdk
For v2.x, in the roadmap: https://wiki.lfaidata.foundation/display/MIL/Milvus+2.0+Roadmap+and+Time+schedule, the C++ sdk is supposed to be implemented in v2.1

**Reply by 852511489** [reply]
Thank you very much！
 I find api in Milvus source code and Bilibili.
